control-plane/csi-driver/proto/node-service.proto

```diff
@@ -20,6 +20,8 @@ service NodePlugin {
   rpc UnfreezeFS (UnfreezeFSRequest) returns (UnfreezeFSReply) {}
   // Find the volume identified by the volume ID, and return volume information.
   rpc FindVolume (FindVolumeRequest) returns (FindVolumeReply) {}
+  // Issue Cleanup for the volume, unmount and disconnect.
+  rpc Cleanup (CleanupRequest) returns (CleanupReply) {}
```

Comment:
how about simply `NodeUnstage` ?

Reply:
We can but it's a little bit different compared to current node unstage.

Reply:
Okay `ForceUnstageVolume` sounds good.